docs/integration/jenkins.md

```diff
@@ -0,0 +1,58 @@
+**jenkins.plugins.with_update** <br>*(long gauge)*      |  The count of plugins in the Jenkins instance that have an newer version reported as available in the current Jenkins update center metadata held by Jenkins. This value is not indicative of an issue with Jenkins but high values can be used as a trigger to review the plugins with updates with a view to seeing whether those updates potentially contain fixes for issues that could be affecting your Jenkins instance
+**jenkins.plugins.inactive** <br>*(long gauge)*      |  The count of plugins in the Jenkins instance that are not currently enabled
+**jenkins.plugins.failed** <br>*(long gauge)*      |  The count of plugins in the Jenkins instance that failed to start. A value other than 0 is typically indicative of a potential issue within the Jenkins installation that will either be solved by explicitly disabling the plugin(s) or by resolving the plugin dependency issues
+**jenkins.executors.free** <br>*(long gauge)*      |  The count of executors available to Jenkins that are not currently in use
+**jenkins.executors.in_use** <br>*(long gauge)*      |  The count of executors available to Jenkins that are currently in use
+**jenkins.runs.success** <br>*(long counter)*      |  The count of job runs which performed succeed
+**jenkins.runs.unstable** <br>*(long counter)*      |  The count of job runs which were unstable
+**jenkins.runs.failure** <br>*(long counter)*      |  The count of job runs which failed
+**jenkins.runs.not_built** <br>*(long counter)*      |  The count of not built job runs
+**jenkins.runs.aborted** <br>*(long counter)*      |  The count of aborted job runs
+**jenkins.jobs** <br>*(long counter)*      |  The total count of jobs
+**jenkins.jobs.scheduled** <br>*(long counter)*      |  The count at which jobs are scheduled. If a job is already in the queue and an identical request for scheduling the job is received then Jenkins will coalesce the two requests. This metric gives a reasonably pure measure of the load requirements of the Jenkins master as it is unaffected by the count of executors available to the system
+**jenkins.jobs.queuing** <br>*(long counter)*      |  The count of queued jobs
+**jenkins.jobs.blocked** <br>*(long counter)*      |  The count at which jobs in the build queue enter the blocked state
+**jenkins.jobs.buildable** <br>*(long counter)*      |  The count at which jobs in the build queue enter the buildable state
+**jenkins.jobs.execution.time** <br>*(long counter)* *(ms)*      |  The amount of time jobs spend in execution state
+**jenkins.jobs.queuing.time** <br>*(long counter)* *(ms)*      |  The total time jobs spend in the build queue
+**jenkins.jobs.blocked.time** <br>*(long counter)* *(ms)*      |  The amount of time jobs in the build queue enter spend in blocked state
+**jenkins.jobs.buildable.time** <br>*(long counter)* *(ms)*      |  The amount of time jobs in the build queue enter spend in buildable state
```

Comment:
The amount of time jobs in the build queue enter spend in buildable state
=>
The amount of time jobs in the build queue spend in buildable state

Reply:
done